### ISSUE TITLE
Notification and Discord Rich Presence reports wrong episode

### ISSUE THREAD

**Issue by georgyorgy1** [erengy/taiga#628]
I'm not exactly good at explaining things so I'll list it down step by step.

1. I would either select Play Next Episode or select an episode (for example, episode 3)
2. Windows would give me a notification that I'm playing episode 12 when I'm actually watching episode 3 (for example)
3. On Discord, it would also report that I'm playing episode 12 even though I'm watching episode 3.

Here's an example filename of what I have on my computer: [AnimeKaizoku] Nisekoi 03 [720p][BD][dedsec].mkv

I'm not sure why it reports the wrong episode but I'll be happy to receive answers/fixes for this. 

Thanks

**Comment by erengy** [comment]
That filename seems okay. I couldn't reproduce the issue with the exact same name.

Here's some questions that might help us figure out what's going on:

- Does this happen when you're watching other series too?
- Do you always get the 12th episode of *Nisekoi* when you're watching the 3rd, or is it a random episode?
- Where is the file located at? Could you copy-paste the entire path?
- Which media player do you use?
- Does your media player display the filename or something else as its window title?
- Is it possible that your media player somehow loads all the files in the same directory (e.g. as a playlist) automatically?

**Comment by georgyorgy1** [comment]
@erengy 

1. I'm not sure. I just started using Taiga recently (2 days ago). The first thing I've watched with Taiga was Blend S episode 11 (HorribleSubs release). I didn't see any problems with it (even with the last episode).

2. Nope, sometimes I would get Episode 8 or 5 or almost any random number even though I picked episode 3. VLC plays the correct episode (the one I picked) but the rich presence and notification would report a different episode.

3. G:\Anime\Nisekoi\Nisekoi is where my Nisekoi folder is (This is Season 1). My anime library points to G:\Anime.

4. I'm using VLC Media Player. VLC prints the title as "Nisekoi 03" instead of the whole thing.

5. VLC prints the title as "Nisekoi 03" instead of the whole filename. 

5. I'm not sure with this one, but I don't think it loads all of the episodes into a playlist. I usually select episodes individually.

**Comment by erengy** [comment]
I downloaded and tested the same files now. This seems to be an issue specific to AnimeKaizoku's *Nisekoi* release. Their chapters are misconfigured, which is why you don't see the OP and the ED while watching the episode (the total length of the 3rd episode is supposed to be 24:10 rather than 21:09).

As for why Taiga detects a random episode number, it's because VLC opens all the files in the directory as I suspected, likely because it's unable to find the OP/ED segments. Other media players I tried (MPC-HC and mpv) didn't have this issue.
